applovin_max/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ## Versions
 
-## x.x.x
+## 3.12.0
```

Comment:
Since we are bumping major versions for native SDKs, this need to be a major version bump as well: 4.0.0

Reply:
Let me fille another PR.